installer/conf/omsagent.conf

```diff
@@ -6,7 +6,7 @@
 <source>
   type exec
   tag heartbeat.output
-  command /opt/microsoft/omsagent/bin/omsadmin.sh -b > /dev/null
+  command /opt/microsoft/omsagent/bin/omsadmin.sh > /dev/null
   format tsv
   keys severity,message
   run_interval 20m
```

Comment:
Shouldn't we remove this entire <source></source> if we aren't using omsadmin.sh for heartbeat anymore?